README.md

```diff
@@ -14,11 +14,13 @@ Project Summary:
 - Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
 - Dynamically rendered created list of tasks in the dedicated placeholder.
 - Created a style.css and set rules for the To-Do List.
+- Added dragged and drop functionality to To-Do list
```

Comment:
* Kindly make your README file more descriptive by adding proper instructions on how to set up and run your project locally. it helps visitors of your repo understand how to run your project locally